src/library_webgpu.js

```diff
@@ -912,7 +912,7 @@ var LibraryWebGPU = {
     var viewFormatCount = {{{ gpu.makeGetU32('descriptor', C_STRUCTS.WGPUTextureDescriptor.viewFormatCount) }}};
     if (viewFormatCount) {
       var viewFormatsPtr = {{{ makeGetValue('descriptor', C_STRUCTS.WGPUTextureDescriptor.viewFormats, '*') }}};
-      desc["viewFormats"] = Array.from({{{ makeHEAPView(`${POINTER_BITS}`, 'viewFormatsPtr', `viewFormatsPtr + viewFormatCount * ${POINTER_SIZE}`) }}},
```

Comment:
Nice find, thank you! Glad to find that these bindings work in wasm64 at all :)

We have the same problem in https://github.com/emscripten-core/emscripten/blob/4765f9c36f3706b37d295627c44307689a88bad7/src/library_webgpu.js#L2501 which takes WGPUFeatureName which is 32-bit.